### ISSUE TITLE
Standardize generated LaTeX markup

### ISSUE THREAD

**Issue by DamienCassou** [pillar-markup/pillar#73]
To get better LaTeX output, we should improve the exported markup:

- we should replace `\begin{code}{}` with `\begin{listing}[language=XXXX]`
- we should replace `\ct` by `\code`

**Comment by DamienCassou** [comment]
cc @cdlm

**Comment by DamienCassou** [comment]
```
Name: Pillar-ExporterLaTeX-DamienCassou.82
Author: DamienCassou
Time: 17 September 2015, 4:48:49.276404 pm
UUID: 78394995-b7cc-4c78-b9c4-b7a1cda59715
Ancestors: Pillar-ExporterLaTeX-DamienCassou.81

Fix #73: Standardize generated LaTeX markup

- replace \begin{code}{} with \begin{listing}[language=XXXX]
- replace \ct by \code
```
